CHANGELOG.md

```diff
@@ -12,8 +12,9 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### Removed
 
 ### Fixed
-
+- Mocha Dependency issue in `opensearch-dsl` gem ([#108](https://github.com/opensearch-project/opensearch-ruby/pull/108))
```

Comment:
Can I nitpick? It should say “Locked mocha …” because it doesn’t say what the issue is otherwise.